Ory/kratosauth.yml

```diff
@@ -27,8 +27,8 @@ services:
     environment:
       - DSN=postgres://kratos:secret@postgres-kratos:5432/kratos?sslmode=disable&max_conns=20&max_idle_conns=4
     ports:
-      - '127.0.0.1:4433:4433'
-      - '127.0.0.1:4434:4434'
+      - '4433:4433'
+      - '4434:4434'
```

Comment:
Please keep as it was here, it's not linked and it's better for security to keep exposed only on localhost.

Reply:
fixed